deploy/charts/dashboards/zfsLocalPV/zfslocalpv.json

```diff
@@ -0,0 +1,1258 @@
+               "spaceLength": 10,
+               "stack": true,
+               "steppedLine": false,
+               "targets": [
+                  {
+                     "expr": "node_zfs_arc_l2_asize{job=\"node-exporter\",instance=~\"$node:$port\"}",
+                     "format": "time_series",
+                     "hide": false,
+                     "interval": "",
+                     "intervalFactor": 2,
+                     "legendFormat": "asize",
+                     "refId": "I",
+                     "step": 2,
+                     "target": ""
+                  },
+                  {
+                     "expr": "node_zfs_arc_l2_hdr_size{job=\"node-exporter\",instance=~\"$node:$port\"}",
+                     "format": "time_series",
+                     "intervalFactor": 2,
+                     "legendFormat": "metadata",
+                     "refId": "D",
+                     "step": 2,
+                     "target": ""
+                  },
+                  {
+                     "expr": "node_zfs_arc_l2_size{job=\"node-exporter\",instance=~\"$node:$port\"}",
+                     "format": "time_series",
+                     "hide": false,
+                     "interval": "",
+                     "intervalFactor": 2,
+                     "legendFormat": "size",
+                     "refId": "B",
+                     "step": 2,
+                     "target": ""
+                  }
+               ],
+               "thresholds": [ ],
+               "timeFrom": null,
+               "timeRegions": [ ],
+               "timeShift": null,
+               "title": "ARC L2 - Size",
+               "tooltip": {
+                  "shared": true,
+                  "sort": 2,
+                  "value_type": "individual"
+               },
+               "type": "graph",
+               "xaxis": {
+                  "buckets": null,
+                  "mode": "time",
+                  "name": null,
+                  "show": true,
+                  "values": [ ]
+               },
+               "yaxes": [
+                  {
+                     "format": "bytes",
+                     "label": "",
+                     "logBase": 1,
+                     "max": null,
+                     "min": null,
+                     "show": true
+                  },
+                  {
+                     "format": "short",
+                     "label": null,
+                     "logBase": 1,
+                     "max": null,
+                     "min": null,
+                     "show": true
+                  }
+               ],
+               "yaxis": {
+                  "align": false,
+                  "alignLevel": null
+               }
+            }
+         ],
+         "title": "ARC L2",
+         "type": "row"
+      }
+   ],
+   "refresh": "10s",
+   "schemaVersion": 27,
+   "style": "dark",
+   "tags": [
+      "OpenEBS",
+      "ZFS-LocalPV"
```

Comment:
Should this be in lower case or upper case?

Reply:
Not sure. For `lvm` pool dashboard, we have used uppercase.